src/components/DrumLoopPlayer.js

```diff
@@ -31,6 +47,16 @@ export default function DrumLoopPlayer({
       </DrumLoopSelect>
     </DrumLoopContainer>
   );
+
+  function handleClick() {
+    startDrumLoop(isPlayin);
+    setIsPlayin(!isPlayin);
+  }
+  function onChange(event) {
+      const currentLoop = event.target.value;
+      getDrumLoop(isPlayin, currentLoop);
+      isPlayin ? setIsPlayin(false) : setIsPlayin(false);
```

Comment:
Please check this here - i dont understand the condition having in both cases the same output